app/queries/update_markdown_content_block_mutator.rb

```diff
@@ -3,7 +3,7 @@ class UpdateMarkdownContentBlockMutator < ApplicationQuery
   include ContentBlockEditable
 
   property :id, validates: { presence: true }
-  property :markdown, validates: { length: { maximum: 10_000 } }
+  property :markdown, validates: { length: { maximum: ContentBlock.markdown_curriculum_editor_max_length } }
```

Comment:
@prashanth726 @hwhelchel Any reason why this is pulled using a class method here and not directly from application secrets, similar to [curriculum_presenter.rb](https://github.com/pupilfirst/pupilfirst/blob/254fa3e2642f8dcc2a368069029093b4bac8e182/app/presenters/schools/courses/curriculum_presenter.rb#L91) ?

Reply:
@mahesh-krishnakumar purely a style choice. It's shorter and more legible to me. Feel free to make a direct call to secrets if you prefer.

Reply:
Okay @hwhelchel Just that the property `markdown_curriculum_editor_max_length` is a configuration for the UI, and it doesn't seem to be associated with the `ContentBlock` model.

Reply:
@mahesh-krishnakumar feel free to change and merge 😃